### PR TITLE
Adds appropriate error messages for 403/404/500 responses

### PR DESCRIPTION
**Pull request by gdpelican**

### CODE REVIEW

**Comment by robguthrie** [comment]
Hey @gdpelican I don' think it's right to change the page the user is on when these events happen. I think a error flash message would be better. Otherwise we are likely to lose their state and stuff.

**Comment by gdpelican** [comment]
We don't change the page they're on at all.. it simply swaps out the
ng_viewport with an error pane.

**Comment by robguthrie** [comment]
So If for some random reason my comment fails to post. How do I get my
comment back?

**Comment by robguthrie** [comment]
Becasue if this swaps out the viewport, I can't use the back button or
anything to get it back.. it's lost. I think it's better to use an error
flash, then the user can copy their text out and do what they need to do.

**Comment by gdpelican** [comment]
That's not covered by this; this only covers the case where visiting a
thread / group page results in a 403 / 404 / 500; we'll have to go through
and come up with sensible failure cases for all of the requests we make,
but this is just the low-hanging fruit of that work.

File: lineman/app/components/thread_page/thread_page_controller.coffee [hunk 1]
```diff
@@ -9,6 +9,8 @@ angular.module('loomioApp').controller 'ThreadPageController', ($routeParams, $l
     @group = @discussion.group()
     $rootScope.$broadcast('viewingThread', @discussion)
     MessageChannelService.subscribeTo("/discussion-#{@discussion.key}", @onMessageReceived)
+  , (error) ->
```
Comment (gdpelican) [inline lineman/app/components/thread_page/thread_page_controller.coffee]: We only broadcast the 'pageError' event when the initial 'Get me the thread / group' request fails. We'll have to broadcast other types of events for other types of failures, I'd imagine.

**Pull request merged**